openwisp_controller/config/templates/admin/config/system_context.html

```diff
@@ -1,8 +1,17 @@
-{% for key, value in system_context.items %}
-    {% if new_line in value %}
-        <p><strong>{{ key }}:</strong><pre>{{ value }}</pre></p>
-    {% else %}
-        <p><strong>{{ key }}:</strong> {{ value }}</p>
-    {% endif %}
-{% endfor %}
+<div> 
+    <div class="system-context">
+        {% if  system_context|length > 0%}
+        {% for key, value in system_context.items %}
+            {% if new_line in value %}
+                <p><strong>{{ key }}:</strong><pre>{{ value }}</pre></p>
+            {% else %}
+                <p><strong>{{ key }}:</strong> {{ value }}</p>
+            {% endif %}
+        {% endfor %}
+        {% else %}
+        <p><strong>There are no system defined variables available right now.</strong></p>
+        {% endif %}
+    </div>
+    <span class="system-context-btn">show</span>
```

Comment:
flag the string as translatable.

Use the `button` tag, eg: `<button class="button system-context">`.
No need to repeat -btn suffix in the css class, let's keep the button styling in the button css class.